script.js

```diff
@@ -1,4 +1,271 @@
+      lastWinner = "you";
+      return nrWinningStatement(
+        compGuess,
+        userGuess,
+        username,
+        userNumWin,
+        totalGuesses,
+        numDraws
+      );
+    } else if (userGuess == "paper" && compGuess == "stone") {
+      userNumWin += 1;
+      lastWinner = "you";
+      return nrWinningStatement(
+        compGuess,
+        userGuess,
+        username,
+        userNumWin,
+        totalGuesses,
+        numDraws
+      );
+    } else if (userGuess == "stone" && compGuess == "scissors") {
+      userNumWin += 1;
+      lastWinner = "you";
+      return nrWinningStatement(
+        compGuess,
+        userGuess,
+        username,
+        userNumWin,
+        totalGuesses,
+        numDraws
+      );
+    } else if (userGuess == compGuess && totalGuesses > 1) {
+      numDraws += 1;
+      return `The computer chose ${compGuess}. <br> You chose ${userGuess}. <br> This is a draw! <br> The last winner of this game was ${lastWinner}, therefore, ${lastWinner} won!`;
+    } else if (userGuess == compGuess && totalGuesses <= 1) {
+      numDraws += 1;
+      return `The computer chose ${compGuess}. <br> You chose ${userGuess}. <br> What are the odds! You got a draw in the first round! <br> There is no winner, please continue!`;
+    } else {
+      lastWinner = "the computer";
+      return `${userGuess} doesn't beat ${compGuess}. <br> Computer: "MUK-JJI-PPA!"`;
+    }
+  }
+};
+
+// Function for computer vs computer game logic
+var playAiGame = function (username) {
+  compGuess1 = guessGenerator();
+  compGuess2 = guessGenerator();
+  // Implement rule: scissors beats paper, paper beats stone, and stone beats scissors. If both parties choose the same object, it's a draw.
+  // No validation here as user just have to click submit for computer guess.
+
+  totalGuesses += 1;
+  if (compGuess1 == "scissors" && compGuess2 == "paper") {
+    userNumWin += 1;
+    return nrWinningStatement(
+      compGuess2,
+      compGuess1,
+      username,
+      userNumWin,
+      totalGuesses,
+      numDraws
+    );
+  } else if (compGuess1 == "paper" && compGuess2 == "stone") {
+    userNumWin += 1;
+    return nrWinningStatement(
+      compGuess2,
+      compGuess1,
+      username,
+      userNumWin,
+      totalGuesses,
+      numDraws
+    );
+  } else if (compGuess1 == "stone" && compGuess2 == "scissors") {
+    userNumWin += 1;
+    return nrWinningStatement(
+      compGuess2,
+      compGuess1,
+      username,
+      userNumWin,
+      totalGuesses,
+      numDraws
+    );
+  } else if (compGuess1 == compGuess2) {
+    numDraws += 1;
+    return `The computer chose ${compGuess2}. <br> You chose ${compGuess1}. <br> This is a draw! <br> Hey ${username}, so far you have won ${userNumWin} of ${totalGuesses} rounds, with ${numDraws} draws! <br> Click "submit" to play another round!`;
+  } else {
+    return `${compGuess1} doesn't beat ${compGuess2}. You lose!`;
+  }
+};
+
 var main = function (input) {
```

Comment:
Great job keeping the main function clean and splitting up the logic handling to other functions!

Code looks clean. Great that functions are named starting with a verb!  

Looking forward to seeing it in action on tuesday!